gc/base/standard/ConcurrentGCIncrementalUpdate.cpp

```diff
@@ -790,19 +790,20 @@ MM_ConcurrentGCIncrementalUpdate::updateTuningStatisticsInternal(MM_EnvironmentB
 	if (_extensions->debugConcurrentMark) {
 		OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
 
-		char pass1Factor[10];
-		char pass2Factor[10];
+		const size_t bufLen = 10;
+		char pass1Factor[bufLen];
+		char pass2Factor[bufLen];
 
 		if (_extensions->cardCleaningPasses > 0) {
-			sprintf(pass1Factor, "%.3f", _cardCleaningFactorPass1);
+			snprintf(pass1Factor, bufLen, "%.3f", _cardCleaningFactorPass1);
```

Comment:
I don't think `snprintf` is available with all compilers (e.g. some older Visual Studio versions). We need to check that it's available in the minimum version supported by OMR (and downstream projects).

Reply:
OpenJ9 uses Visual Studio 2017 as a minimum, so that project should have no problem with this change.
I couldn't find anything in OMR itself that specifies a minimum version, but I did find this code in [omrsig_internal.hpp](https://github.com/eclipse/omr/blob/master/omrsigcompat/omrsig_internal.hpp#L88-L105) that anticipates support for several older versions.

Reply:
I found this line in doc/CodingStandard.md:

```
sprintf() should not be used. Use the omrstr_printf() function from the port library, instead.
```

Reply:
I updated the code by using `omrstr_printf()` in ConcurrentGCIncrementalUpdate.cpp.